### ISSUE TITLE
Sorting of the videos is somewhat weird, can it be on date of uploading or the date of the upload on the mediaplatform itself?

### ISSUE THREAD

**Issue by w1g0** [ethereumclassic/ethereumclassic.github.io#940]

**Comment by IstoraMandiri** [comment]
I was trying to keep things simple by having a single `date` field that was the same as whatever was shown on youtube.

I can see that in some cases, when we want to add a video that has an old upload date, then it make the added video not very visible. 

 There is no easy fix for this other than creating an `added` field with the date.

I guess the front page should list newest videos by `added`, and then the videos page lists by `date` ?

**Comment by TheCrowbill** [comment]
I agree.  Track the `date added` to do a front page splash of newest videos.

I suggest, however, that the video page lists be sorted by `creator `and alphabetized by `title`.

Would this add too many parameters?

**Comment by IstoraMandiri** [comment]
I disagree about the sorting on the videos page, because it will bias exposure to results. 

I feel like creation date is more relevant and keeps the content fresh.

**Comment by w1g0** [comment]
> I guess the front page should list newest videos by `added`, and then the videos page lists by `date` ?

Sounds good to me 👍
